keyboards/ducky/one2mini/readme.md

```diff
@@ -0,0 +1,15 @@
+# one2mini
```

Comment:
Could we get some flashing instructions here?

Reply:
Yes, as soon as the flashing stuff is safe! I don't want to trust the general public with directions regarding the chip manufacturer's flash util - it's very easy to flash away your bootloader with that thing and then you'll have to dig out an SWD/OpenOCD capable setup to fix it all!
Happy to help testers in the Discord in the meantime (basically until next week, when the flash util is completely sorted).

Reply:
Well, could you link the instructions in the comments here, so we can take a look, in the meanwhile.